test/AutoRest.TestServer.Tests/Infrastructure/TestServerV1.cs

```diff
@@ -25,13 +25,15 @@ public TestServerV1()
         {
             var portPhrase = "Started server on port ";
             var startup = Path.Combine(GetBaseDirectory(), "dist", "cli", "cli.js");
+            var coverageDirectory = Path.Combine(GetArtifactsDirectory(), "coverage");
```

Comment:
Does the test server create a directory when it doesn't exist?

Reply:
yes, it does.